docs/getting-started/raster/raster_step-by-step.md

```diff
@@ -341,6 +341,48 @@ Replace all `<>` place holders with the real values that we got from all previou
 ```
 - Note - **extent** taken from step 2 - where bbox is calculated.
 
+### OpenLayers (1.x)
+```javascript
+  import L from "leaflet";
+...
+...
+...
+
+    const parser = (url) =>{
+      return wmtsUrl.replace("maps", "{s}")
+    .replace("{Style}", '<LAYER_STYLE>')                              // from Step_4
+    .replace("{TileMatrixSet}", '<LAYER_TILE_MATRIX_SET_ID>')         // from Step_4
+    .replace("{TileMatrix}", "{z}")
+    .replace("{TileRow}", "{y}")
+    .replace("{TileCol}", "{x}");
+    }
+
+    const bounds = L.latLngBounds([
+      [extent[1], extent[0]],                                         // from Step_2
+      [extent[3], extent[2]],                                         // from Step_2
+    ]);
+
+    const url = '<LAYER_WMTS_URL>'                                    // from Step_3 or Step_4
+    const parsedUrl = parser(url)                                     
+
+    const layer = L.tileLayer(parsedUrl + 'token=${RASTER_TOKEN}',{   // received RASTER auth token
```

Comment:
add the following remarks:
// in case of 'header' authentication 
//  headers: { 'X-API-KEY': RASTER_TOKEN },    (update to leaflet terminology)   // received RASTER auth token
add remark on "in case of queryParameter authentication" above this row

Reply:
added a link for example extension for adding header token